templates/layouts/_base.html

```diff
@@ -94,10 +94,10 @@
 
 
 {% block bodyStart %}
-  {% if google_tag_manager_id and google_tag_manager_auth and google_tag_manager_preview  %}
+  {% if google_tag_manager_id and google_tag_manager_auth  %}
     <!-- Google Tag Manager (noscript) -->
     <noscript>
-      <iframe src="https://www.googletagmanager.com/ns.html?id={{ google_tag_manager_id }}&gtm_auth={{ google_tag_manager_auth }}&gtm_preview={{ google_tag_manager_preview }}&gtm_cookies_win=x"
+      <iframe src="https://www.googletagmanager.com/ns.html?id={{ google_tag_manager_id }}&gtm_auth={{ google_tag_manager_auth }}&&gtm_cookies_win=x"
```

Comment:
This was not (but could have been) removed in the previous PR as it was not part the testing.